public/variables.css

```diff
@@ -40,9 +40,66 @@
   --chart-timeout: #d9a620;
   --chart-unresolved: #d9a620;
 
+  --syntax-comment: var(--neutral-300);
+  --syntax-punctuation: var(--neutral-800);
+  --syntax-keyword: var(--neutral-400);
+  --syntax-class: var(--neutral-600);
+  --syntax-boolean: var(--neutral-400);
+  --syntax-symbol: var(--neutral-700);
+  --syntax-number: var(--neutral-800);
+  --syntax-string: var(--neutral-400);
+  --syntax-variable: var(--neutral-100);
+  --syntax-operator: var(--neutral-900);
+  --syntax-function: var(--neutral-600);
+  --syntax-background: var(--neutral-50);
+  --syntax-text: var(--neutral-800);
```

Comment:
**issue:** This is going to invert the colors in dark and light mode respectively. Have you tested it against both backgrounds?

**suggestion:** [The theme](https://github.com/motform/stimmung-themes) I suggested we take inspiration from quite clearly states "following the maxim that a theme that highlights everything paradoxically highlights nothing". The theme is quite well thought out and tested, so my suggestion is to simply implement it as is. I am afraid that we, by simply trying to imitate it, will just make it worse.
Another alternative, which means less work, is to simply use a ready-made theme for prism.